<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trade</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/trade.ico</normaloff>:/icons/trade.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_add">
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_delete">
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/del.png</normaloff>:/icons/del.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textFilterStock">
          <property name="text">
           <string>Отбор по складу:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterStock"/>
        </item>
        <item>
         <widget class="QPushButton" name="resetFilter">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/reset.png</normaloff>:/icons/reset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Database">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="action_nomenclature"/>
    <addaction name="action_stock"/>
    <addaction name="action_purchase"/>
    <addaction name="action_sale"/>
    <addaction name="action_return_sale"/>
    <addaction name="action_remnants"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_CheckUpdate"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_Database"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_CheckUpdate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/update.png</normaloff>:/icons/update.png</iconset>
   </property>
   <property name="text">
    <string>Проверить обновления</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_nomenclature">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/directory.png</normaloff>:/icons/directory.png</iconset>
   </property>
   <property name="text">
    <string>Номенклатура</string>
   </property>
  </action>
  <action name="action_stock">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/directory.png</normaloff>:/icons/directory.png</iconset>
   </property>
   <property name="text">
    <string>Склады</string>
   </property>
  </action>
  <action name="action_remnants">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/report.png</normaloff>:/icons/report.png</iconset>
   </property>
   <property name="text">
    <string>Остатки на складах</string>
   </property>
  </action>
  <action name="action_purchase">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/doc.png</normaloff>:/icons/doc.png</iconset>
   </property>
   <property name="text">
    <string>Закупки</string>
   </property>
  </action>
  <action name="action_sale">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/doc.png</normaloff>:/icons/doc.png</iconset>
   </property>
   <property name="text">
    <string>Продажи</string>
   </property>
  </action>
  <action name="action_return_sale">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/doc.png</normaloff>:/icons/doc.png</iconset>
   </property>
   <property name="text">
    <string>Возвраты от покупателей</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
